lib/word_formatter.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'hangman_rules'
+
+class WordFormatter
+
+  attr_reader :rules
+
+  def initialize(rules)
+    @rules = rules
+  end
+
+  def format(secret_word)
+    formatted_word = secret_word.split("").map do |letter|
+      letter = replaced_letter(letter)
+    end
+    formatted_word.join("")
+  end
+
+  private
+
+  def replaced_letter(letter)
+    if not_guessed?(letter)
+      letter = "_"
```

Comment:
Same here. You can write something as simple as:
```
if not_guessed?(letter)
  "_"
else
  letter
end
```

Reply:
True